### ISSUE TITLE
Interpretation of ambiguous tokens as timezone names

### ISSUE THREAD

**Issue by ghost** [mattermost/mattermost-plugin-walltime#53]
Certain text tokens (words) may be misinterpreted as timezones by the plugin under some circumstances.

More precisely, an issue was experienced and reported against [Collabora's currently-deployed version of the plugin](https://gitlab.collabora.com/tools/mattermost/mattermost-plugin-walltime/-/tree/d00bde4efd968d29bcc4854dcd188a292867f264).

In particular, the phrase:

```
People visiting Buñol towards the end of August get a good chance to participate in La Tomatina (under normal circumstances)
```

... was reformatted to ...

```
People visiting Buñol towards the end of `August get` *(Thu, Aug 1, 2019 1:00 AM PDT)* a good chance to participate in La Tomatina (under normal circumstances)
```

This is due to the token `get` being misinterpreted by the plugin as [`GET` (Georgia Standard Time)](https://time.is/GET).

**Comment by jayaddison-collabora** [comment]
Note: this issue does _not_ appear replicable against [published release v1.0.0 of the plugin](https://github.com/mattermost/mattermost-plugin-walltime/releases/tag/1.0.0) (so this is likely a regression).

**Comment by jayaddison-collabora** [comment]
From bisecting the cause of the regression: this bug was introduced by b92064221070e2564aa9eba2ddbbcb4beebe70f7 - "Update to chrono-node v2.3.4 (#40)".
